virtcontainers/proxy.go

```diff
@@ -161,3 +171,90 @@ type proxy interface {
 	//check if the proxy has watched the vm console.
 	consoleWatched() bool
 }
+
+func (p *proxyBuiltin) watchConsole(proto, console string, logger *logrus.Entry) (err error) {
+	var (
+		scanner *bufio.Scanner
+		conn    net.Conn
+	)
+
+	switch proto {
+	case consoleProtoUnix:
+		conn, err = net.Dial("unix", console)
+		if err != nil {
+			return err
+		}
+		// TODO: add pty console support for kvmtools
```

Comment:
could you please file an issue and mention it here?

Reply:
Hi @devimc Done.

Reply:
thanks